invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -129,7 +135,11 @@ def get_affiliation(self, obj):
                     )
 
                     if id_scheme == "ROR":
-                        identifier_value = "https://ror.org/" + identifier["identifier"]
+                        identifier_value = identifier["identifier"]
+                    elif detected_scheme:
+                        identifier_value = to_url(
+                            identifier["identifier"], identifier["scheme"]
+                        )
```

Comment:
minor: sort of same thing as the comment above here, we have to be careful which identifiers we convert to URLs and which ones we leave as is (by default I would not convert identifiers to URLs).

Unfortunately, there's no docs or anything related to that, it's all coming from experience with sending metadata to the DataCite REST API and seeing what gets accepted or not...

---

This will probably affect how the DCAT XSLT serializer behaves, since big pat of it relies on having the full URLs in order to determine schemes, etc. If that's the case let's discuss IRL how to address this (probably will have to be done in the DCAT serializer explicitly)